packages/instant/src/components/zero_ex_instant_container.tsx

```diff
@@ -64,6 +68,17 @@ export class ZeroExInstantContainer extends React.Component<{}, ZeroExInstantCon
                         </SlidingPanel>
                         <CurrentStandardSlidingPanel />
                     </Container>
+                    <Container
+                        display={{ sm: 'none', default: 'block' }}
+                        marginTop="10px"
+                        marginLeft="auto"
+                        marginRight="auto"
+                        width="140px"
+                    >
+                        <Text href={ZERO_EX_SITE_URL}>
```

Comment:
Are we using the text for the hover styles too? Otherwise an anchor might just suffice.

Reply:
We're not using the hover style.  It felt kind of unnatural to use `Text` here but wasn't sure the accepted pattern.

Reply:
Changed to `a`